### PR TITLE
VS-1341 Improve test cost calculation

### PR DESCRIPTION
**Pull request by gbggrant**
This PR fixes an error in the cost calculation in that we were not handling pre-emption. That is, a shard could get preempted and thus we'd have multiple cost measurements for that shard. 
I've addressed this by looking for a single 'representative' instance of each shard. But despite this, I found there is still non-determinism in the values written to the cost table. So, I've reset the thresholds we use for calculating if the test 'passes'. 
We may need some time to deal with finding real numbers (that is, my threshold may now be too conservative).

Passing test runs:
All Chromosomes [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/adc09aff-e6bc-4289-ac34-d74b86eb92e2).
Chromosome 20/X/Y [here](https://app.terra.bio/#workspaces/gvs-dev/GVS%20Integration/job_history/1b9a7d34-6d4f-42ca-8434-924ca84ee0ed).

### CODE REVIEW

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl [hunk 1]
```diff
@@ -286,8 +286,8 @@ task AssertCostIsTrackedAndExpected {
 
         echo "project_id = ~{project_id}" > ~/.bigqueryrc
         bq --apilog=false query --project_id=~{project_id} --format=csv --use_legacy_sql=false \
-            'SELECT call, step, event_key, sum(event_bytes)
-              FROM `~{dataset_name}.cost_observability`
+            'SELECT call, step, event_key, sum(event_bytes) FROM
+                (SELECT distinct step, call, event_key, event_bytes, shard_identifier FROM `~{dataset_name}.cost_observability`)
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl:290]: The `distinct` is treating `event_bytes` like it's a component of a composite key?

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl [hunk 2]
```diff
@@ -286,8 +286,8 @@ task AssertCostIsTrackedAndExpected {
...
               GROUP BY call, step, event_key
               ORDER BY call, step, event_key' > output/cost_observability.csv
```
Comment (mcovarr) [inline scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl:292]: tl;dr I don't understand what this query is trying to do, happy to discuss faces in standup. It seems that the `GROUP BY` is squashing out the notion of shards?

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl (hunk 1 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl]: This was an error. I have rewritten the query.

File: scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl (hunk 2 above)
Comment (gbggrant) [reply scripts/variantstore/wdl/GvsQuickstartVcfIntegration.wdl]: Query is rewritten. Closing this thread.

**Review by mcovarr — APPROVED** [review]
👍 I believe the query now works as intended, tolerances can be adjusted if needed

**Pull request merged**